src/main/g8/ddl-scripts/create_tables.cql

```diff
@@ -0,0 +1,89 @@
+
+-- This CQL script will create the keyspace and all tables needed for the this sample. 
+-- It includes the messages and snapshot tables (write-side) and the projection offset_store table (read-side).
+
+-- NOTE: the keyspace as created here is probably not what you need in a production environment. 
+-- This is good enough for local development though. 
+
+CREATE KEYSPACE IF NOT EXISTS shoppingcart
+  WITH REPLICATION = { 'class' : 'SimpleStrategy','replication_factor':1 };
+
+USE shoppingcart;
```

Comment:
this should be derived from the name given to g8. Let's use
```
"$name;format="word-only,lowercase"$"
```

that becomes `shoppingcartservice`

Reply:
correct

Reply:
Done, without the quotes around it.